src/js/chartAuthoring.js

```diff
@@ -237,6 +360,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             $(this).find(dataEntryLabelSelector).val(currentData.label).trigger("change");
             $(this).find(dataEntryValueSelector).val(currentData.value).trigger("change");
         });
+
+        that.events.onUpdateDataEntryPanel.fire();
+    };
+
+    floe.chartAuthoring.addSonificationControlListeners = function(that) {
```

Comment:
My first instinct at seeing "add...Listeners" was to add event listeners. What do you think of names like "addSonificationButtonHandler" or "addSonificationControllerHandler"?